doc/team.rst

```diff
@@ -0,0 +1,124 @@
+.. title:: Team
+
+PyGMT Team Gallery
+==================
+
+We are an international team dedicated to building a Pythonic API for the
+Generic Mapping Tools (GMT). Our goal is to improve GMT's accessibility for
+new and experienced users by creating user-friendly interfaces with the GMT
+C API, supporting rich display in Jupyter notebooks, and integrating with
+the PyData ecosystem.
+
+All our welcome to become involved with the PyGMT project! For more information
+about how to get involved, see the :doc:`contributing`.
+
+Distinguished Contributors
```

Comment:
On my mac (13 inch), the display of **Leonardo** is not perfect for 100% view, while 90% is okay. Meanwhile, my workstation is okay since it has a monitor of 23 inch.

100% | 90%
--- | ---
<img width="928" alt="100" src="https://user-images.githubusercontent.com/50591376/122893716-29292d00-d379-11eb-881c-3092875c302a.png"> | <img width="828" alt="90" src="https://user-images.githubusercontent.com/50591376/122893802-3c3bfd00-d379-11eb-9c4d-81b603bb8dce.png">

Reply:
Thanks for pointing this out. I updated the css style, but unfortunately I cannot figure out how to make the items both a nice grid and also flexible to fit the contents. Please let me know if you think the new style is better.

Reply:
I have the same issue as **Dongdong** pointed in https://github.com/GenericMappingTools/pygmt/pull/1308#issuecomment-867300192 now.